### PR TITLE
Alignment

### PR DESCRIPTION
**Pull request by tyxxxx**
r

It would be strange if the text were the same size as the picture, now they are aligned in the center of a line.

### CODE REVIEW

**Comment by BarneyShi** [comment]
@tyxxxx Nice. Just a suggestion: we'd better remove the bullets before every use pic coz it's kinda

**Review by BarneyShi — APPROVED** [review]

**Comment by tyxxxx** [comment]
屏幕快照 2021-06-10 下午6 33 05
like this one?
In addition, I have a suggestion that I think we can make the post component bigger (such as the screenshot I uploaded above). Now, the post component is smaller and looks a little empty
屏幕快照 2021-06-10 下午6 36 29

**Comment by parksj23** [comment]
@tyxxxx Regarding your suggestion about the posts container being a bit small, if you pull the project_2 branch as it is now you'll see that it shows 3 posts per row. I'm not sure why it's only showing 2 posts per row on your branch. Is that something you could look at before you merge?

**Comment by NaithanB** [comment]
The issue with the PostCollection being small is because it was moved into a container in App.js. In App.css, I set the PostCollection component to be 80% of the parent when the screen is over 600px wide. Moving it into the container makes it 80% of the size of the container rather than the main page. It's super easy to fix. We just need to change line 87 in App.css from "width: 80%;" to "width: 100%;"

**Comment by NaithanB** [comment]
Hi @tyxxxx, I still see the bulletpoints for each user in the user list but it looks like you removed them based on the screenshot. Have you committed that update?

**Comment by tyxxxx** [comment]
Yeah @NaithanB @parksj23 I pushed that update! you just need approve this commit. Then I can merge into Project_2

**Comment by parksj23** [comment]
Hi Yuxin, I just changed one thing and pushed the code to this branch.

**Comment by NaithanB** [comment]
Thanks for doing that @parksj23! That makes the layout of the components look correct now.

**Review by NaithanB — APPROVED** [review]

**Review by parksj23 — APPROVED** [review]

**Comment by NaithanB** [comment]
Oh hey, @BarneyShi, are you still able to access your detail component from this branch? I don't think I can on my machine.

**Review by NaithanB — CHANGES_REQUESTED** [review]
Oh wait, I see. The user list component is still visible when we view a post detail page and it pushes the post detail component below it. Should we hide the user list when we view a post detail page?

**Comment by NaithanB** [comment]
Oh wait, I see. The user list component is still visible when we view a post detail page and it pushes the post detail component below it. Should we hide the user list when we view a post detail page?

**Comment by BarneyShi** [comment]
@NaithanB No worries. Just need to put the `container` inside route path `/`.

**Comment by tyxxxx** [comment]
I think it's better to hide all interface If you want to hide the user list, Either hide them together or make them visible together. what do u think?

File: vanhouse/src/App.js [hunk 1]
```diff
@@ -11,9 +12,16 @@ function App() {
...
+                <Container fluid>
+                    <Row>
+                        <Col lg={3} md={3}><UserList /></Col>
+                        <Col>               
                 <Route exact path="/">
                     <PostCollection/>
                 </Route>
+                </Col>
+                    </Row>
+                </Container>
```
Comment (BarneyShi) [inline vanhouse/src/App.js:24]: Here I believe if you change it to
```
<Route exact path="/">
    <Container fluid>
                    <Row>
                        <Col lg={3} md={3}><UserList /></Col>
                        <Col>               

                    <PostCollection/>

                </Col>
                    </Row>
                </Container>
</Route>
```
It'll be alright then.

Comment (BarneyShi) [reply vanhouse/src/App.js:24]: @tyxxxx

**Comment by parksj23** [comment]
Naithan also pointed out one thing I could fix in my css that would fix the issues we found with the checkboxes in the 'Add post' modal. Is it okay if I push my code right now?

**Comment by BarneyShi** [comment]
> Naithan also pointed out one thing I could fix in my css that would fix the issues we found with the checkboxes in the 'Add post' modal. Is it okay if I push my code right now?

How do you think about directly modifying code on `project_2`?

**Comment by parksj23** [comment]
> > Naithan also pointed out one thing I could fix in my css that would fix the issues we found with the checkboxes in the 'Add post' modal. Is it okay if I push my code right now?
> 
> @parksj23 How do you think about directly modifying code on `project_2`?

Okay, I'll go do that right now. Just a sec!

**Comment by parksj23** [comment]
> > > Naithan also pointed out one thing I could fix in my css that would fix the issues we found with the checkboxes in the 'Add post' modal. Is it okay if I push my code right now?
> > 
> > 
> > @parksj23 How do you think about directly modifying code on `project_2`?
> 
> Okay, I'll go do that right now. Just a sec!

Pushed my modifications to project_2!

**Comment by tyxxxx** [comment]
Also, Can I merge this into Project_2?

**Comment by BarneyShi** [comment]
I think it's good to go. How do you guys think about? @NaithanB @parksj23

**Comment by parksj23** [comment]
Yeah I think it looks okay. The title (my part) looks wonky in the post detail page so I'll try to fix that today or tomorrow but otherwise it looks great!

**Comment by tyxxxx** [comment]
@NaithanB

**Comment by parksj23** [comment]
> @NaithanB

He might not get to this until tomorrow actually.

**Comment by BarneyShi** [comment]
@tyxxxx Let's merge it and see how it goes.

**Comment by tyxxxx** [comment]
merge

**Pull request merged**